radio_beam/tests/test_beam.py

```diff
@@ -24,33 +26,43 @@ def test_classic_header():
     # Instantiate from header
     fname = data_path("NGC0925.bima.mmom0.fits.gz")
     hdr = fits.getheader(fname)
-    bima_beam_file = radio_beam.Beam.from_fits_header(fname)
-    bima_beam_hdr = radio_beam.Beam.from_fits_header(hdr)
+    bima_beam_file = Beam.from_fits_header(fname)
+
+    npt.assert_equal(bima_beam_file.major.value, hdr["BMAJ"])
+    npt.assert_equal(bima_beam_file.minor.value, hdr["BMIN"])
+    npt.assert_equal(bima_beam_file.pa.value, hdr["BPA"])
+
+    bima_beam_hdr = Beam.from_fits_header(hdr)
+
+    npt.assert_equal(bima_beam_hdr.major.value, hdr["BMAJ"])
+    npt.assert_equal(bima_beam_hdr.minor.value, hdr["BMIN"])
+    npt.assert_equal(bima_beam_hdr.pa.value, hdr["BPA"])
+
 
 def test_from_aips_test():
     aips_fname = data_path("ngc0925_na.fits.gz")
     aips_hdr = fits.getheader(aips_fname)
-    aips_beam_hdr = radio_beam.Beam.from_fits_header(aips_hdr)
-    np.testing.assert_almost_equal(aips_beam_hdr.sr.value, 9.029858054819811e-10)
-    aips_beam_file = radio_beam.Beam.from_fits_header(aips_fname)
-    np.testing.assert_almost_equal(aips_beam_file.sr.value, 9.029858054819811e-10)
+    aips_beam_hdr = Beam.from_fits_header(aips_hdr)
+    npt.assert_almost_equal(aips_beam_hdr.sr.value, 9.029858054819811e-10)
+    aips_beam_file = Beam.from_fits_header(aips_fname)
+    npt.assert_almost_equal(aips_beam_file.sr.value, 9.029858054819811e-10)
 
-def test_from_casa_test():
+def test_fits_from_casa_test():
```

Comment:
It was probably my fault originally, but I think we can remove the trailing `test` unless we want to start the department of redundancy department